dnf-behave-tests/createrepo_c/steps/repodata.py

```diff
@@ -69,6 +105,12 @@ def repodata_are_consistent(context, path):
             filelists_path = decompressed_p
         elif "other" == item.name:
             other_path = decompressed_p
+        elif "primary_zck" == item.name:
+            prim_zck_path = decompressed_p
+        elif "filelists_zck" == item.name:
+            filelists_zck_path = decompressed_p
+        elif "other_zck" == item.name:
+            other_zck_path = decompressed_p
```

Comment:
Hmm, the convention is to have the variable part on the left and constant part on the right when doing comparisons, I find this strangely twisted :slightly_smiling_face: 

The code is already there, so up to you if you wanna flip all the conditions (don't flip just those that are added, that would just make it worse...)